client/src/components/SurveyFab/SurveyFab.module.scss.d.ts

```diff
@@ -0,0 +1,12 @@
+declare namespace SurveyFabNamespace {
```

Comment:
Just curious, what does Fab in SurveyFab stand for?

Reply:
FAB is a Floating Action Button :)